components/Layout/Layout.styled.js

```diff
@@ -1,14 +1,24 @@
 import styled from "styled-components";
 
-const AppContainer = styled.div`
+const ContentContainer = styled.div`
+  height: fit-content;
   position: relative;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
   border: solid green;
-  padding: 0 10px;
-  gap: 10;
+  padding: 20px 0;
 `;
 
-export { AppContainer };
+const Footer = styled.div`
```

Comment:
```suggestion
const Footer = styled.footer`
```

Reply:
- [x] Already fixed forgot to push it. Thanks for the reminder. 👍